SEP-0011.md

```diff
@@ -0,0 +1,154 @@
+
+Because of the need to follow these format guidelines, ``sunpy`` developers must pay close attention to formatting, constantly making small decisions about code formatting based on rules, on taste, or on a mix of both.
+While some of PEP-8 is focused on guidelines like naming conventions that cannot be automated, much of it is essentially algorithmic and hence these decisions are a needless burden on the developers.
+
+With **automated code formatting**, the workload of code formatting is done by computers, saving human bandwidth for higher value and complexity activities like the logic or interface design.
+Collectively, contributors no longer need to think about code formatting when writing or in code reviews.
+By ceding control to  *Black* they can focus on the content of the code and not worry about details like where to break a line or how to arrange a list.
+Whatever the formatter does is the expected result, by definition.
+
+Automated code formatting will increase consistency across the code base of every SunPy package.
+Currently the style of each package reflects the preferences of the maintainers.
+For instance, ``sunpy`` uses ``autopep8`` whereas every other package already uses *Black*.
+There are other more subtle differences and SunPy contributors find this confusing.
+This has come up in the past.
+
+While there are multiple code formatters for Python, the *Black* Python code formatter produces good enough results that it is getting significant traction in many open source projects, including: scikit-learn, pytest, tox, Pyramid, Django, Hypothesis, SQLAlchemy, virtualenv, pandas, Pillow, and Twisted.
+While adoption by this list of high-profile community-based open source projects does not automatically imply that it is a good fit for SunPy, it demonstrates unequivocally several points:
+
+- Technical issues related to the initial transition and subsequent maintenance of *Black* formatting are surmountable.
+  There is plenty of prior art and good documentation.
+- Sociological issues related to developer preferences are surmountable.
+  Evidence from other projects is that developers either embrace or at least accept autoformatting.
+- Developer documentation can provide clear instructions to contributors on how to implement *Black* formatting in their development process.
+  Issues with this process are no more difficult (and arguably easier) to resolve than enforcing PEP-8 standards.
+
+Therefore, this SEP proposes to adopt *Black* for the SunPy Project.
+
+#### Concerns and mitigations
+
+##### Foolish consistency
+
+A concern that has been highlighted by some members of the community (including, initially, the author of this SEP, and highlighted in detail in [this blog post](https://luminousmen.com/post/my-unpopular-opinion-about-black-code-formatter) is that adopting *Black* violates the PEP-8 principle that "A Foolish Consistency is the Hobgoblin of Little Minds".
```

Comment:
you really need to edit this to not just be APE 20 :stuck_out_tongue:

Reply:
I can delete this?

Reply:
Removed.